test-suite/shared/src/test/require-jdk7/org/scalajs/testsuite/javalib/util/logging/HandlerTest.scala

```diff
@@ -0,0 +1,40 @@
+package org.scalajs.testsuite.javalib.util.logging
+
+import java.util.logging._
+
+import org.junit.Test
+import org.junit.Assert._
+
+class HandlerTest {
+  class TestHandler extends Handler {
+    override def flush(): Unit = {}
+
+    override def publish(record: LogRecord): Unit = {}
+
+    override def close(): Unit = {}
+  }
+
+  class TestFilter extends Filter {
+    override def isLoggable(record: LogRecord): Boolean = true
```

Comment:
Should also test when it is not loggable, I suggest adding a `logable: Boolean` parameter to the constructor and writing some tests for when it is false.